### DISCUSSION TITLE
Dynamic Template Placeholders

### DISCUSSION THREAD

**Discussion by dfoss2022** [outline/outline#6970] [General] [answered]
Is there a way to do dynamic place holders for templates? Things like `{{date}}:{{time}}` to get the current date and time when a note is created with the template, `{{title}}` to add in the title of the note when created.

**Comment by tommoor** [comment, marked as answer]
`{date}`, `{time}` and `{datetime}` will work, it's a very hidden feature

**Reply by IamTaoChen** [reply]
It can only work on the title

**Reply by intech** [reply]
Yes, it only works in title :(

**Reply by tommoor** [reply]
I can't reproduce this issue, make sure you're running an uptodate version – or it could be that the variables are not replaced in some specific cirumstance so you could provide an example

**Reply by T2brozz** [reply]
I'm also not getting it to work. I tried `{author}` , ``` `{author }` ``` and both also as placeholder. But nothing replaced it with my name when I create a document with this template

**Reply by tommoor** [reply]
It only works when creating a document from a template, is that what you're doing?

**Reply by T2brozz** [reply]
Yeah in My collection in a subfodler , I click on the +  then in the top right I select the template from the dropdown

**Reply by T2brozz** [reply]
did I do something wrong?

**Reply by tommoor** [reply]
Nope, it sounds like a bug

**Reply by tommoor** [reply]
Fix incoming: https://github.com/outline/outline/pull/8044
